src/__tests__/extensions/surveys.test.ts

```diff
@@ -552,74 +552,6 @@ describe('SurveyManager', () => {
         })
     })
 
-    describe('canRenderSurvey', () => {
```

Comment:
tests removed as I moved that method to `posthog-surveys` to stop duplicating rendering logic